### DISCUSSION TITLE
launch in debug mode in iOS 14+

### DISCUSSION THREAD

**Discussion by trey-a-hope** [codemagic-ci-cd/codemagic-docs#2369] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Build an iOS app in debug mode.

### Expected results

I should be able to open my ipa file that was built in debug mode.

### Actual results

That app crashes instantly after opening.

I was curious if the people at Codemagic are privy to this issue.

_To launch in debug mode in iOS 14+, run flutter run from Flutter tools, run from an IDE with a Flutter IDE plugin or run the iOS project from Xcode.
Alternatively profile and release mode apps can be launched from the home screen._

**Comment by dtrdic** [comment]
Note: conversation continued on Discord support channel
